src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const sum = days * PRICE;
```

Comment:
The variable 'sum' could have a more descriptive name, such as 'totalCost', to better reflect what it represents.